src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
```

Comment:
Don't use static methods in your solution. Instead of having a static Random instance, consider creating a private non-static instance of Random within the class.